libs/zedUpload/sftputil/sftp.go

```diff
@@ -193,7 +193,7 @@ func ExecCmd(cmd, host, user, pass, remoteFile, localFile string,
 				}
 			}
 		}
-		return stats
```

Comment:
Missing "reach" or "get" in comment. But isn't there some golang/lint annotation which can be used for notreached?

Reply:
Fixed it..